File: hellocharts/src/main/java/net/raquezha/lecho/hellocharts/listener/LineChartOnValueSelectListener.java
package net.raquezha.lecho.hellocharts.listener;


import net.raquezha.lecho.hellocharts.model.PointValue;

public interface LineChartOnValueSelectListener extends OnValueDeselectListener {

    void onValueSelected(int lineIndex, int pointIndex, PointValue value);

}

File: hellocharts/src/main/java/lecho/lib/hellocharts/listener/LineChartOnValueSelectListener.kt
package lecho.lib.hellocharts.listener

import lecho.lib.hellocharts.model.PointValue
import lecho.lib.hellocharts.model.TouchCoordinates

interface LineChartOnValueSelectListener : OnValueDeselectListener {

    fun onValueSelected(
        lineIndex: Int,
        pointIndex: Int,
        value: PointValue,
        touchCoordinates: TouchCoordinates?
    )
}